### PR TITLE
L2S-3549 - King's birthday 2024 exception update

### PR DESCRIPTION
**Pull request by MossJTanda**
Fixed broken syntax from .yaml file for ease of use.

### CODE REVIEW

**Comment by MossJTanda** [comment]
@benaitcheson

File: au.yaml [hunk 1]
```diff
@@ -517,7 +517,7 @@ tests:
     expect:
       name: "Queen's Birthday"
   - given:
-      date: ['2024-09-23', '2025-09-30']
+      date: ['2025-09-30']
```
Comment (benaitcheson) [inline au.yaml:520]: How come this one was changed? It's still relevant for 2024?

Comment (MossJTanda) [reply au.yaml:520]: I keep reading it in reverse, will restore the test

**Review by benaitcheson — APPROVED** [review]
Looks good. Let's not merge this until we've tested in payaus, this way we don't have to make a fourth PR.

**Review by benaitcheson — APPROVED** [review]
Looks good! We can tackle what happens in 2026 when Fairwork release the date.

**Review by benaitcheson — APPROVED** [review]

**Pull request merged**